src/Kunstmaan/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -108,7 +108,8 @@ public function getConfigTreeBuilder()
                         ->integerNode('min_length')->defaultNull()->end()
                         ->integerNode('max_length')->defaultNull()->end()
                     ->end()
-            ->end();
+            ->end()
+            ->booleanNode('hide_sidebar')->defaultFalse()->end();
```

Comment:
Doesn't this need to be flipped? First the new option and than the closing end method?
```suggestion
                ->booleanNode('hide_sidebar')->defaultFalse()->end()
            ->end();
```

Reply:
The end is from password_restrictions